boards/esp32-ethernet-kit/Makefile.features

```diff
@@ -0,0 +1,13 @@
+CPU_MODEL = esp32-wrover
+
+# common board and CPU features
+include $(RIOTBOARD)/common/esp32/Makefile.features
+
+# additional features provided by the board
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_pwm
+
```

Comment:
To get `cpu/esp_common/periph/spi.c` compiled, feature `periph_spi` has to be provided.
```suggestion

ifneq (,$(filter esp_jtag,$(USEMODULE)))
  FEATURES_PROVIDED += periph_spi
endif

```